swf/core.py

```diff
@@ -56,3 +59,14 @@ def __init__(self, *args, **kwargs):
             raise ValueError('invalid region: {}'.format(self.region))
 
         logger.debug("initiated connection to region={}".format(self.region))
+
+        self.hostname = socket.gethostname()
+
+    def send_endpoint_usage(endpoint, host="grafana.botify.com", port=8089):
```

Comment:
Shouldn't be hardcoded, please push force with no botify value asap, since we're disclosing a private info on a public repository here ;-) You have examples in the repo of using simpleflow settings. You can grep for "SIMPLEFLOW_ENABLE_DISK_CACHE" for instance to get a simple enough example.